code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -11,9 +11,6 @@ proc/issyndicate(mob/living/M as mob)
 	required_enemies = 5
 	recommended_enemies = 5
 
-	uplink_welcome = "Corporate Backed Uplink Console:"
-	uplink_uses = 120

```

Comment:
Yeah, this is olddd from when Nukies had a *single* sole uplink.

Now they exact get individual uplinks with individual assignments and the re-assignment console has the rest of them.